src/config/lenker.ts

```diff
@@ -1,8 +1,8 @@
 enum lenker {
   Home = '/',
-  TokenFornyet = '/token-fornyet',
-  Bulkinnsending = '/batchinnsending/krav',
-  OversiktKrav = '/oversikt/krav'
+  TokenFornyet = '/:language/token-fornyet',
```

Comment:
Her mister du vel bakoverkompabiliteten? Kunne hatt dobbelt opp eller redirect for å støtte de som allerede har laget bokmerke eller kopiert lenka?